src/drivers/intel/baytrail/ipc.c

```diff
@@ -125,7 +125,6 @@ static void irq_handler(void *arg)
 					isr, shim_read(SHIM_IMRD),
 					shim_read(SHIM_IPCXH));
 		} else {
-			_ipc->host_msg = msg;
```

Comment:
@dbaluta if you remove this, there's no need for the msg variable too?

Reply:
@ranj063, true. Unless, we want to keep the debugging message consistent. 

`		
                  msg = shim_read(SHIM_IPCXL);

		/* TODO: place message in Q and process later */
		/* It's not Q ATM, may overwrite */
		if (_ipc->host_pending) {
			trace_ipc_error("ipc: dropping msg 0x%x", msg);
			trace_ipc_error(" isr 0x%x imrd 0x%x ipcxh 0x%x",
					isr, shim_read(SHIM_IMRD),
					shim_read(SHIM_IPCXH));
`

But I guess, msg will always be 0.

Reply:
@dbaluta yes. I think its redundant. So we should just remove the trace logs and msg altogether.

Reply:
@ranj063, so do you mean something like this.

https://pastebin.com/xsFYtwXv

Sorry for using pastebin but inserting code here just kills me.

So, the best thing to do this would be to attach another patch to this series, where we really explain why in case of _ipc->host_pending == true we just do nothing and also do not print any error message.

Reply:
@dbaluta the second trace error message is still useful to indicate if we do drop a message. I was only suggesting to remove the msg part of it because it is not really useful.

Reply:
@ranj063 ok, it's clear now. will respin.